### PR TITLE
Better logging

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

File: lib/src/generate/options.dart [hunk 1]
```diff
@@ -57,9 +57,11 @@ class BuildOptions {
       } else {
         color = _red;
       }
-      var message = '$color[${record.level}]$_endColor ${record.loggerName}: '
+      var message = '${Platform.isWindows ? '' : '\x1b[2K\r'}'
```
Comment (kegluneq) [inline lib/src/generate/options.dart]: Just curiosity, what's this craziness do?

**Comment by kegluneq** [comment]
LGTM. What does it mean that "info logs now overwrite each other by default"?

File: lib/src/generate/options.dart (hunk 1 above)
Comment (jakemac53) [inline lib/src/generate/options.dart]: Oh ha, it deletes the entire previous line and sets the cursor back to the beginning. Seems to be the easiest way.

**Comment by jakemac53** [comment]
They don't cause a new line in your console, they just overwrite the current line. This just helps to get rid of the spammy logs and only highlight warnings/errors.

This only happens with the default logger implementation (which is a console logger).

**Pull request merged**